Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/outrun/ios-application-strings/) for OutRun/iOS-application-strings.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/outrun/-/ios-application-strings/horizontal-auto.svg)

aaaberg commented:
@timfraedrich, I made some changes to most of the Swedish strings for accuracy. Thanks for this app, great job!

lpla commented:
I also modified "Apple Health" naming in Spanish and Catalan, so it is consistent with other sport apps and the official iPhone online manual.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Polish)

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Polish)

Currently translated at 100.0% (268 of 268 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Leopoldo Pla Sempere <leopoldoplasempere@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/pl/
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-strings/pl/
Translation: OutRun/iOS-application-changelog
Translation: OutRun/iOS-application-strings

weblate committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (267 of 267 strings)

Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Lawrence of Arabia <lawrenceofarabia@outlook.it>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-strings/it/
Translation: OutRun/iOS-application-changelog
Translation: OutRun/iOS-application-strings

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translation: OutRun/iOS-application-changelog

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translation: OutRun/iOS-application-changelog

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translation: OutRun/iOS-application-changelog

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translation: OutRun/iOS-application-changelog

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (268 of 268 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Leopoldo Pla Sempere <leopoldoplasempere@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/es/
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-strings/es/
Translation: OutRun/iOS-application-changelog
Translation: OutRun/iOS-application-strings

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translation: OutRun/iOS-application-changelog

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translation: OutRun/iOS-application-changelog

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translation: OutRun/iOS-application-changelog

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translated using Weblate (Catalan)

Currently translated at 100.0% (2 of 2 strings)

Translated using Weblate (Catalan)

Currently translated at 100.0% (268 of 268 strings)

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Leopoldo Pla Sempere <leopoldoplasempere@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-changelog/ca/
Translate-URL: https://hosted.weblate.org/projects/outrun/ios-application-strings/ca/
Translation: OutRun/iOS-application-changelog
Translation: OutRun/iOS-application-strings
